diktat-analysis.yml

```diff
@@ -0,0 +1,230 @@
+- name: DIKTAT_COMMON
+  enabled: true
+  configuration:
+    isCopyrightMandatory: true
+    domainName: please enter your domain name
+    copyrightText: please enter your copyright text
+- name: CLASS_NAME_INCORRECT
+  enabled: true
+  configuration: {}
```

Comment:
Is `{}` parsed as an empty map in yml too? Or can this field just be left empty?

Reply:
This field can't be left empty, because it cannot parse when configuration is empty

Reply:
OK